control-plane/csi-driver/controller/src/main.rs

```diff
@@ -95,6 +96,9 @@ async fn main() -> anyhow::Result<()> {
         CsiControllerConfig::get_config().rest_endpoint()
     );
 
+    //Starts PV Garbage collector in a separate thread.
```

Comment:
s/thread/task
though the comment seems redundant as the code is easy to read that it is doing that.

Reply:
Should i just comment it as Starts PV Garbage collector??

Reply:
Incorporated